src/scripts/discord.js

```diff
@@ -25,22 +25,22 @@ const observer = (event, arg) => {
       rpc.setActivity({
         ...idleStatus,
         ...{
-          details: `Listening to ${mediaInfoModule.mediaInfo.title}`,
+          details: `${mediaInfoModule.mediaInfo.title}`,
           state: mediaInfoModule.mediaInfo.artist
             ? mediaInfoModule.mediaInfo.artist
             : "unknown artist(s)",
           startTimestamp: parseInt(now),
           endTimestamp: parseInt(remaining),
           largeImageKey: mediaInfoModule.mediaInfo.image,
           largeImageText: (mediaInfoModule.mediaInfo.album) ? mediaInfoModule.mediaInfo.album : `${idleStatus.largeImageText}`,
-          buttons: [{ label: "Play on Tidal", url: mediaInfoModule.mediaInfo.url }],
+          buttons: [{ label: "Listen Along", url: mediaInfoModule.mediaInfo.url }],
```

Comment:
Great addition.

Reply:
> I don't see the point of changing this to "Listen Along" as this would be misleading since all it does is link to the page on tidal's website

It's more simple, though, and takes up less space on the rich presence, plus TIDAL is already mentioned on the rich presence, so it's kinda redundant.

Reply:
> > I don't see the point of changing this to "Listen Along" as this would be misleading since all it does is link to the page on tidal's website
> 
> It's more simple, though, and takes up less space on the rich presence, plus TIDAL is already mentioned on the rich presence, so it's kinda redundant.

It still ends up being misleading also TIDAL already being mentioned is not a reason to call it redundant as the actual Spotify integration also still says "Play on Spotify" even though Discord shows that you are listening to stuff on Spotify